SPECS/tdnf/tdnf.spec

```diff
@@ -96,11 +96,18 @@ Requires:       python3
 %description python
 python bindings for tdnf
 
+%package        autoupdate
```

Comment:
New version of tdnf includes two services dedicated to automating updates. I came up with the name of the package on my own, feel free to modify.